dataprofiler/__init__.py

```diff
@@ -1,3 +1,5 @@
+import logging
```

Comment:
More that I think about it, should be in a new file bc we shouldn't have dp.logging